test/Functional/Tests/GenerationTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+namespace Hostnet\Component\EntityPlugin\Functional;
+
+use Composer\Autoload\AutoloadGenerator;
+use Composer\Composer;
+use Composer\Config;
+use Composer\EventDispatcher\EventDispatcher;
+use Composer\IO\BufferIO;
+use Composer\Package\RootPackage;
+use Composer\Repository\InstalledArrayRepository;
+use Composer\Repository\RepositoryManager;
+use Hostnet\Component\EntityPlugin\Plugin;
+use RecursiveDirectoryIterator;
+use RecursiveIteratorIterator;
+use Symfony\Component\Console\Output\StreamOutput;
+
+class GenerationTest extends \PHPUnit_Framework_TestCase
```

Comment:
Coverage indicates nothing but lines being hit in tests. If this is a test, why lie about the lines you hit? Your test could fail on a line which seems to not get hit in your test.